manifests/plugin.pp

```diff
@@ -22,16 +22,23 @@
   $config_content  = undef,
   $update_url      = undef,
   $plugin_dir      = '/var/lib/jenkins/plugins',
-  $username        = 'jenkins',
-  $group           = 'jenkins',
   $enabled         = true,
```

Comment:
This would be an API breaking change.  I think the parameters need to be retained.